src/Command/BakeSimpleMigrationCommand.php

```diff
@@ -65,9 +67,9 @@ public function fileName($name): string
      */
     public function getPath(Arguments $args): string
     {
-        $path = ROOT . DS . $this->pathFragment;
+        $path = ROOT . DS . $this->pathFragment . DS . $args->getOption('source') . DS;
         if ($this->plugin) {
-            $path = $this->_pluginPath($this->plugin) . $this->pathFragment;
+            $path = $this->_pluginPath($this->plugin) . $this->pathFragment . DS . $args->getOption('source') . DS;
```

Comment:
``$args->getOption('source')`` could be extracted into an intermediate variable to improve readability and reduce redunant calls.